src/Util/ApiLoader.php

```diff
@@ -134,7 +134,7 @@ public function getClientOptions()
      */
     protected function getFromCache()
     {
-        $cacheKey = $this->getCacheKey();
+        $cacheKey = $this->getCacheKey() ?? '';
```

Comment:
A blank cache key doesn't make any sense? Why is this returning a blank key?

Reply:
`CacheInterface::get()`: Argument #1 ($key) must be of type string, null given, called in `silverstripe-maintenance/src/Util/ApiLoader.php` on line 138

Reply:
That sounds like a bug?  A null/blank cache key is not a proper cache key.

I'm not sure how that happens because the 2 class methods that implement abstract ApiLoader::getCacheKey() are:

**ModuleHealthLoader**
```php
    protected function getCacheKey()
    {
        return sha1(json_encode($this->getModuleNames()));
    }
```

`echo sha1('')` = `da39a3ee5e6b4b0d3255bfef95601890afd80709` - so that can't return null

and

**SupportedAddonsLoader**
```php
    protected function getCacheKey()
    {
        return 'addons';
    }
```

Can you replicate this issue locally?

Reply:
DONE